pandas/core/generic.py

```diff
@@ -5740,6 +5741,134 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def convert_dtypes(
+        self: FrameOrSeries,
+        infer_objects: bool_t = True,
+        convert_string: bool_t = True,
+        convert_integer: bool_t = True,
+        convert_boolean: bool_t = True,
+    ) -> FrameOrSeries:
+        """
+        Convert columns to best possible dtypes, optionally using dtypes supporting
+        ``pd.NA``.
```

Comment:
I find the "optionally" a bit confusing, as that sounds that it is "opt-in", while it is actually the default behaviour. So I would remove the "optionally" here (which can also ensure the summary line fits on a single line, as needed to follow the docstring standards). And then in an extended summary below we can specify that you can opt-out for the different dtypes via keywords.